PowerEditor/installer/nativeLang/portuguese.xml

```diff
@@ -397,7 +397,7 @@ Nota de tradução:
 					<Item id="50010" name="Dica anterior dos parâmetros da função"/>
 					<Item id="50011" name="Dica seguinte dos parâmetros da função"/>
 					<Item id="50005" name="Alternar para gravação de macro"/>
-					<Item id="50006" name="Preenchimento da localização"/>
+					<Item id="50006" name="Preenchimento da caminho"/>
```

Comment:
@horus68 @donho "Preenchimento `do` caminho" not `da`

Reply:
Apart from that little typo, LGTM. Thanks @donho for notifying me!
@horus68 Thank you for improving the grammar of the translation. I just ask that the next time you want to edit/update the translation you notify me here or by e-mail **(I mention this at the end of the `portuguese.xml` file, you probably haven't seen it).**

Reply:
@hugok79 
Thank you for your review.

There are currently several changes for the localization files:
* Add "Close to system tray" in MISC preference (035ef19)
* Improve GUI for commands for the system tray in Preferences (cd45afc)
* Avoid user confusion between Global override & Default Styles (1890ea6)
* Fix Find dialog status bar wrong messaging (9010147)
* Fix wrong messages for replacement status (bfe835e)

Will you gonna work on them? If yes, I will merge directly this PR, then you just fix it in your PR.
Is it OK for you?

Reply:
Yes, I've noticed those new changes!